app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "What pension does a teacher get?"
+heading: "What pension does a teacher get?"
```

Comment:
Is this the right title/h1? We are not really answering the question? (Does it need to be a question?)